src/admin/components/Fields.vue

```diff
@@ -1,316 +1,338 @@
                         <div class="dokan-input-group">
-                            <span class="dokan-input-group-addon">{{ __( 'Day', 'dokan-lite' ) }}</span>
-                            <select v-if="!fieldData.grouped" class="regular" :name="sectionId + '[' + fieldData.name + '][days]'" :id="sectionId + '[' + fieldData.name + ']'" v-model="fieldValue[fieldData.name]['days']">
+                            <span class="dokan-input-group-addon">{{ __( 'Fourth Quarter', 'dokan-lite' ) }}</span>
+                            <select v-if="!fieldData.grouped" class="regular" disabled v-model="disbursementSettings.quarterly.fourth">
                                 <option v-if="fieldData.placeholder" value="" v-html="fieldData.placeholder"></option>
-                                <option v-if="!( 'L' !== fieldValue[fieldData.name]['week'] && ( 'saturday' === optionKey || 'sunday'=== optionKey ) )" v-for="( optionVal, optionKey ) in fieldData.options.days" :value="optionKey" v-html="optionVal"></option>
+                                <option v-for="( optionVal, optionKey ) in fieldData.options.fourth" :value="optionKey" v-html="optionVal"></option>
                             </select>
                         </div>
                     </div>
+                    <div class="dokan-schedule-week-day-container">
+                        <div class="col-3">
+                            <div class="dokan-input-group">
+                                <span class="dokan-input-group-addon">{{ __( 'Week', 'dokan-lite' ) }}</span>
+                                <select v-if="!fieldData.grouped" class="regular" :name="sectionId + '[' + fieldData.name + '][week]'" :id="sectionId + '[' + fieldData.name + ']'" v-model="fieldValue[fieldData.name]['week']">
+                                    <option v-if="fieldData.placeholder" value="" v-html="fieldData.placeholder"></option>
+                                    <option v-for="( optionVal, optionKey ) in fieldData.options.week" :value="optionKey" v-html="optionVal"></option>
+                                </select>
+                            </div>
+                        </div>
+                        <div class="col-3">
+                            <div class="dokan-input-group">
+                                <span class="dokan-input-group-addon">{{ __( 'Day', 'dokan-lite' ) }}</span>
+                                <select v-if="!fieldData.grouped" class="regular" :name="sectionId + '[' + fieldData.name + '][days]'" :id="sectionId + '[' + fieldData.name + ']'" v-model="fieldValue[fieldData.name]['days']">
+                                    <option v-if="fieldData.placeholder" value="" v-html="fieldData.placeholder"></option>
+                                    <option v-if="!( 'L' !== fieldValue[fieldData.name]['week'] && ( 'saturday' === optionKey || 'sunday'=== optionKey ) )" v-for="( optionVal, optionKey ) in fieldData.options.days" :value="optionKey" v-html="optionVal"></option>
```

Comment:
Never use `v-if` and `v-for` inside same element. Wrap it with a `template` element and use `v-if` inside that.
Apply this everywhere the `v-if` and `v-for` used inside the same element.